### ISSUE TITLE
Handle servers which serve ODF files with gzip compression

### ISSUE THREAD

**Issue by kossebau** [kogmbh/WebODF#858]
When doing the XHR to load the ODF file, gzip responses from the server should be prevented it seems. Both because ODF files are zip archives already, unless flat, and because the code does not handle that (from what I understood so far, more investigation needed).

For now the workaround is to configure the server to not use gzip compression for ODT files. But that affects all request for ODF files, not just from WebODF, and ideally WebODF itself would exactly tell what encoding it wants in the XHR.

**Comment by alecpl** [comment]
I'd definitely like to see a solution that works without disabling the compression.

**Comment by kossebau** [comment]
@alecpl "without disabling" means exactly what? if the XHR to fetch the document does not list "gzip" as supported, are there servers which get into trouble, because they want to server everything gzipped?

**Comment by alecpl** [comment]
Could not webodf handle compressed files?

**Comment by kossebau** [comment]
Yes, webodf ideally could also handle that :) but until that is implemented, it seems to me a quicker fix for now is to not claim to support gzip content. Do you see any principle problem with that?
